.drone.yml

```diff
@@ -6,34 +6,54 @@ services:
   web:
     image: fpfis/httpd-php-dev:7.1
     environment:
-     - DOCUMENT_ROOT=/test/oe_profile
+    - DOCUMENT_ROOT=/test/oe_profile
```

Comment:
Shouldn't you indent these lines?

Reply:
They are both valid YAML, it will be anyway considered as an array under `environment`